src/common/contracts/core/campaigns/client.ts

```
@@ -49,23 +49,23 @@ export const update_campaign = ({
 export const delete_campaign = ({ args }: { args: { campaign_id: CampaignId } }) => {
-  return contractApi.call("delete_campaign", {
+  return contractApi.call<{}, Campaign>("delete_campaign", {
     args,
     deposit: floatToYoctoNear(0.021),
     gas: FULL_TGAS,
   });
 };
 
 export const donate = (args: DirectCampaignDonationArgs, depositAmountYocto: IndivisibleUnits) =>
-  contractApi.call("donate", {
+  contractApi.call<{}, CampaignDonation>("donate", {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)